src/containers/App.tsx

```diff
@@ -235,7 +238,11 @@ const Content = (): JSX.Element => {
                             })}
                         >
                             <ToastProvider>
-                                <Header />
+                                {settings?.dashboard === 'Poster' ? (
```

Comment:
Ikke relevant for denne PR'en, men vi burde kanskje se på om vi skal gi hver Route ansvar for om man skal vise Header, Navbar, etc. Er litt mange caser for når, hvor og hva de skal vise.